kubernetes/chart/zulip/values.yaml

```diff
@@ -0,0 +1,225 @@
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+# -- Custom labels to add to the Zulip StatefulSet.
+statefulSetLabels: {}
+# -- Custom annotations to add to the Zulip StatefulSet.
+statefulSetAnnotations: {}
+
+# -- Custom labels to add to the Zulip Pod.
+podLabels: {}
+# -- Custom annotations to add to the Zulip Pod.
+podAnnotations: {}
+
+# -- Can be used to override the default PodSecurityContext (fsGroup, runAsUser
+# and runAsGroup) of the Zulip _Pod_.
+podSecurityContext: {}
+  # fsGroup: 1000
+  # runAsUser: 1000
+  # runAsGroup: 1000
+
+# -- Can be used to override the default SecurityContext of the Zulip _container_.
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+# -- Service type and port for the Kubernetes service that connects to Zulip.
+# Default: ClusterIP, needs an Ingress to be used. On Minikube, use a "NodePort"
+# to circumvent the need for an Ingress.
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  # -- Enable this to use an Ingress to reach the Zulip service.
+  enabled: false
+  # -- Can be used to add custom Ingress annotations.
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    # -- Host for the Ingress. Should be the same as
+    # `zulip.environment.SETTING_EXTERNAL_HOST`.
+    - host: zulip.example.com
+      # -- Serves Zulip root of the chosen host domain.
+      paths:
+      - path: /
+  # -- Set a specific secret to read the TLS certificate from. If you use
+  # cert-manager, it will save the TLS secret here. If you do not, you need to
+  # manually create a secret with your TLS certificate.
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - zulip.example.com
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+autoscaling:
+  enabled: false
+  minReplicas: 1
+  maxReplicas: 100
+  targetCPUUtilizationPercentage: 80
+  # targetMemoryUtilizationPercentage: 80
+
+# -- Optionally add a nodeSelector to the Zulip pod, so it runs on a specific
+# node.
+# Ref: https://kubernetes.io/docs/user-guide/node-selection/
+nodeSelector: {}
+
+# -- Tolerations for pod assignment.
+# Ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+tolerations: []
+
+# -- Affinity for pod assignment.
+# Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+affinity: {}
```

Comment:
is the docker image currently made multiplatform? e.g. arm? if not maybe we should consider adding a default affinity for those architectures

Reply:
IMO this shouldn't block merging this PR. If you think it should, please suggest what affinity to put in and I'll do it. Or add a commit (IIRC I allowed the maintainer to push to this PR)

Reply:
I am happy to make a PR when this is merged